docs/modules/ROOT/pages/guide-keep3r.adoc

```diff
@@ -0,0 +1,138 @@
+# Setting up a keep3r
+
+This guide will walk you through the process of running a Keeper in the https://keep3r.network/[keep3r.network] by leveraging Defender xref:relay.adoc[Relays] and xref:autotasks.adoc[Autotasks].
+
+WARNING: The keep3r.network project is in beta. Use at your own risk.
+
+[[what-is-a-keeper]]
+## What is a Keeper?
+
+In the context of the keep3r network, Keepers are participants who execute available jobs in exchange for a reward. A contracts may register a job to be executed, and once it's available, a registered Keeper can send a transaction to execute it.
+
+[quote, 'https://docs.keep3r.network/#keepers[keep3r.network documentation]']
+____
+A Keeper is the term used to refer to an external person and/or team that executes a job. This can be as simplistic as calling a transaction, or as complex as requiring extensive off-chain logic. The scope of Keep3r network is not to manage these jobs themselves, but to allow contracts to register as jobs for keepers, and keepers to register themselves as available to perform jobs. It is up to the individual keeper to set up their devops and infrastructure and create their own rules based on what transactions they deem profitable.
```

Comment:
This is not formatted as a quote.  

@frangio how do we format this as a quote?

Reply:
The syntax is correct. The docs site is not styling it as a quote. Will take a look at that.

Reply:
Should be better the next time the deploy preview is rendered.